server/server.js

```diff
@@ -32,6 +32,10 @@ function setUpStaticFolders() {
   app.use(rootPath +"css", express.static(cssFolder))
   var jsFolder = path.join(staticFolder, 'js')
   app.use(rootPath +"js", express.static(jsFolder))
+  var mapFontsFolder = path.join(staticFolder, 'map', 'fonts')
+  app.use(rootPath +"mapFonts", express.static(mapFontsFolder, {setHeaders: function(res){res.setHeader("Content-Encoding","gzip")}}))
+  var mapFolder = path.join(staticFolder, 'map')
+  app.use(rootPath +"map", express.static(mapFolder))
```

Comment:
We don't use jslint, but I think that one line is pretty long and that there should be whitespace around operators.